vite.config.shared.ts

```diff
@@ -3,7 +3,7 @@
 /**
  * Shared Vite config settings for all components
  */
-import { defineConfig } from 'vite'
+import { defineConfig, loadEnv } from 'vite'
```

Comment:
There are other changes required in this file, such as [this line](https://github.com/Kong/shared-ui-components/blob/c2dcf89302f1159af9f65417204bc8035b042f19/vite.config.shared.ts#L109) that allows utilizing the root `.env` file(s)

Reply:
We also want to externalize the `i18n` package for all packages like we do on these two lines:
- https://github.com/Kong/shared-ui-components/blob/c2dcf89302f1159af9f65417204bc8035b042f19/vite.config.shared.ts#L72
- https://github.com/Kong/shared-ui-components/blob/c2dcf89302f1159af9f65417204bc8035b042f19/vite.config.shared.ts#L79

If added to this shared config, we first need to ensure it does **not** impact the actual `i18n` package itself.

Reply:
Done. How do we make sure the `i18n` package is not affected? I ran `pnpm build` and it looks fine.

Reply:
Easiest way would be to check the bundle analyzer for the package before and after adding this setting

Reply:
OK, here's the result:
- For `@kong-ui-public/i18n`, the bundle analyzer outputs are identical before and after adding this setting.
- For `@kong-ui-public/entities-shared`, the `i18n` chunk is in the output before adding this setting and disappears after.

LGTM.